src/box/lua/config/instance_config.lua

```diff
@@ -2029,30 +1736,10 @@ return schema.new('instance_config', schema.record({
             'off',
         }, {
             default = 'auto',
-            validate = function(data, w)
-                local scope = w.schema.computed.annotations.scope
-                if data == nil or scope == nil then
-                    return
-                end
-                if scope ~= 'global' then
-                    w.error('sharding.rebalancer_enabled must be defined in ' ..
-                            'the global scope.')
-                end
-            end,
         }),
         rebalancer_disbalance_threshold = schema.scalar({
             type = 'number',
             default = 1,
-            validate = function(data, w)
-                local scope = w.schema.computed.annotations.scope
-                if data == nil or scope == nil then
-                    return
-                end
-                if scope ~= 'global' then
-                    w.error('sharding.rebalancer_disbalance_threshold should '..
-                            'be a defined in global scope')
-                end
-            end,
         }),
         rebalancer_max_receiving = schema.scalar({
```

Comment:
There are a couple of options in the `sharding` section, where the validation function remains in `instance_config.lua`. For example, `sharding.rebalancer_max_receiving`.

Reply:
Thank you! I missed it.